pages/services/data-science-engine/1.0.0/overview/index.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: layout.pug
+navigationTitle: Overview
+excerpt: Overview of DC/OS Data Science Engine features
+title: Overview
+menuWeight: 1
+model: /services/data-science-engine/data.yml
+render: mustache
+---
+
+#  Overview
+
+{{ model.techName }} supports and develops the interactive computing products Jupyter Notebook and Jupyter Hub. 
+
+## The Jupyter Family
+
+The data science community has converged around Jupyter Notebook; an interactive computing notebook developed and supported by Project Jupyter, as the de facto user interface for cloud computing. Jupyter Notebook has proven itself to be a phenomenally effective tool for prototyping and exploration. 
+{{ model.techName }} is the next-generation interface for Project Jupyter, offering the same tools that are familiar to Jupyter Notebook users: terminal, text editor, notebook, file browser, and so forth. Users can work with text editors, terminals, data file viewers, and other components in a notebook with tabbed work areas.
+## Why {{ model.techName }}? 
+{{ model.techName }} enables data science teams to run their compatible Notebooks on scalable Cloud-Native cluster setting, with secured and controlled access to massive data sets, all while keeping onboarding time and operation costs down. Bringing these properties to the native Jupyter stack makes enterprise adoption of Jupyter Notebooks seamless. And will set the data science team on an accelerated path to deliver their Machine Learning projects. 
```

Comment:
The first sentence is confusing. 
```
enables data science teams to run their compatible Notebooks on scalable Cloud-Native cluster setting
```
run notebooks on scalable settings? 

Perhaps we mean run notebooks on scalable cluster resources? Please clarify.

Reply:
It's not necessary anyway. It reads more like marketing material. I'm taking it out.